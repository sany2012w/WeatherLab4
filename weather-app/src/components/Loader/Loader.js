import React from 'react';

class Loader extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div className="loader">Loading...</div>
    );
  }
}

export default Loader;
