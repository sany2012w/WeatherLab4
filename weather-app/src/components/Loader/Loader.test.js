import React from "react";
import Loader from "./Loader";
import renderer from "react-test-renderer";

it(`Loader displayed correctly`, () => {
  const tree = renderer.create(<Loader />).toJSON();
  expect(tree).toMatchSnapshot();
});
