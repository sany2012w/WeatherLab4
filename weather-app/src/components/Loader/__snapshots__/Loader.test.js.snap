// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader displayed correctly 1`] = `
<div
  className="loader"
>
  Loading...
</div>
`;
